instrumentation/metric/histogram.go

```diff
@@ -16,24 +16,24 @@ type Histogram struct {
 
 type histogramExport struct {
 	Name    string
-	Min     int64
-	P50     int64
-	P95     int64
-	P99     int64
-	Max     int64
+	Min     float64
+	P50     float64
+	P95     float64
+	P99     float64
+	Max     float64
 	Avg     float64
 	Samples int64
 }
 
 func newHistogram(name string, max int64) *Histogram {
 	return &Histogram{
 		namedMetric: namedMetric{name: name},
-		histo:       hdrhistogram.NewWindowed(5, max, 1, 3),
+		histo:       hdrhistogram.NewWindowed(5, 0, max, 1),
```

Comment:
ok, I see that you fixed a bug where min was set to max, but why change sigfigs to 1?